docs/pages/management/dynamic-resources/terraform-provider.mdx

```diff
@@ -83,66 +81,61 @@ To prepare credentials for a local Teleport user:
            - access_list
            verbs: ['list','create','read','update','delete']
    version: v7
-   ---
-   kind: user
-   metadata:
-     name: terraform
-   spec:
-     roles: ['terraform']
-   version: v2
    ```
 
-   These settings configure a user and role named `terraform` with the permissions
+   These settings configure a role named `terraform` with the permissions
    required to manage resources in your Teleport cluster.
 
-1. Create the `terraform` user and role by running the following command:
+1. Create the `terraform` role by running the following command:
 
    ```code
-   $ tctl create -f terraform.yaml
+   $ tctl create -f terraform_role.yaml
    role 'terraform' has been created
-   user "terraform" has been created
    ```
 
-   The `terraform` user can't sign in to get credentials, so you must have another user
-   **impersonate** the `terraform` account to request a certificate.
+1. Create a new file called `create_and_run_terraform_bot.sh`, open it in an editor, and
```

Comment:
I think it would make the guide a little simpler to turn the shell scripts into commands that the reader would paste directly into the terminal at the appropriate pointa in the guide, similar to the way we do it in other docs guides. That way, they wouldn't need to manage files and file permissions.

**Edit:** After finishing the guide, I saw the note that you can re-run the two bot scripts to start/stop the bot. If we're keeping the shell scripts, maybe we could add this clarification earlier in the guide?

Reply:
Added context here. Yeah, this is just a [not friendly process](https://gravitational.slack.com/archives/C08HF8E9F/p1707942263917899), unfortunately.